projects/addon-mobile/directives/elastic-sticky/elastic-sticky.service.ts

```diff
@@ -35,6 +43,7 @@ export class TuiElasticStickyService extends Observable<number> {
                                     0,
                                 ),
                             ),
+                            tuiZoneOptimized(this.zone),
```

Comment:
Should we add `distinctUntilChanged`? Otherwise it works like just a zonefull stream

Reply:
done